examples/hello/flow.py

```diff
@@ -1,14 +1,16 @@
-from metaflow import FlowSpec, step, torchrun, current, kubernetes
-
-N_NODES = 2
+from metaflow import FlowSpec, step, kubernetes, environment, torchrun, current
 
 
 class HelloTorchrun(FlowSpec):
     @step
     def start(self):
-        self.next(self.torch_multinode, num_parallel=N_NODES)
+        self.next(self.torch_multinode, num_parallel=2)
 
-    @kubernetes(image="docker.io/pytorch/pytorch:latest", cpu=2)
+    @kubernetes(
+        image="registry.hub.docker.com/pytorch/pytorch:2.5.1-cuda12.4-cudnn9-runtime",
+        cpu=2,
+    )
+    @environment(vars={"OMP_NUM_THREADS": 1})
```

Comment:
Can we add a comment on why we need this in a hello world example

Reply:
yeah, it is to remove a warning, will add the comment

Reply:
Make sure to mention that it's not really needed for this and it's only to suppress the warning :)